phc-rails/app/models/service_list.rb

```diff
@@ -0,0 +1,16 @@
+# This table has a single row for each unique service
+
+class ServiceList < ActiveRecord::Base
+    def self.services
+      self.all.map(&:salesforce_name)
```

Comment:
use `.pluck` instead.
`self.all.map` will load all records from the database into memory and then loop through and grab the salesforce_name, while `self.pluck(&:salesforce_name)` will only load the values for that column from the database into memory.